capstone-project/src/main/webapp/map/map.component.js

```diff
@@ -71,12 +72,24 @@ angular.module('map').component('mapComponent', {
             return containerDiv;
         }
 
+        // Sends a marker to the backend for saving.
+        function postMarker(marker) {
+
+            const markerJson = JSON.stringify(marker);
+            const data = { marker: markerJson }
+            $http({
```

Comment:
Why not use the $http.post method? 
https://docs.angularjs.org/api/ng/service/$http#post